### ISSUE TITLE
this._parentForm does not always have "fire"

### ISSUE THREAD

**Issue by nathanjohnson320** [PolymerElements/iron-form-element-behavior#14]
I am manually creating and removing custom elements and when I do this such as in the below code the parentForm does not have the "fire" function.

```
      this.element = document.createElement(elementName);

      // Clear the existing element and put the new one in
      var child = this.$.createdElement.childNodes[0];
      if (typeof child !== 'undefined') {
        Polymer.dom(this.$.createdElement).removeChild(child);
      }

      Polymer.dom(this.$.createdElement).appendChild(this.element);
```

When the element is removed the _parentForm element does not have a "fire" function and throws an error.

**Comment by notwaldorf** [comment]
I think this was fixed in the latest release (1.0.3). Would you mind pulling the latest code and giving it another go, please? Thanks!

**Comment by nathanjohnson320** [comment]
That fixed it, thanks!
